### PR TITLE
remove airbyte-oauth

### PR DESCRIPTION
**Pull request by c-p-b**
## What
removes the airbyte-oauth module

## How
remove usages in persistence layer

### CODE REVIEW

File: airbyte-oauth/build.gradle [hunk 1]
```diff
@@ -8,7 +8,6 @@ dependencies {
     implementation project(':airbyte-config:config-persistence')
     implementation project(':airbyte-json-validation')
     implementation libs.airbyte.protocol
-    testImplementation project(':airbyte-oauth')
```
Comment (pedroslopez) [inline airbyte-oauth/build.gradle:11]: Should this file be removed entirely?

**Review by pedroslopez — APPROVED** [review]
👋

Do you think we should run a /test for a java destination just in case? Or does the connectors-base build passing cover it?

**Review by pedroslopez — COMMENTED** [review]
If the build passes again, :shipit: !

**Comment by cpdeethree** [comment]
> 👋
> 
> Do you think we should run a /test for a java destination just in case? Or does the connectors-base build passing cover it?

I would think the destinationacceptancetest and the connectoracceptancetest suites would cover this, or am I wrong? cc @evantahler

**Pull request merged**